packages/google_maps_flutter/lib/src/ui.dart

```diff
@@ -125,6 +126,9 @@ class GoogleMapOptions {
   /// True if the map view should respond to zoom gestures.
   final bool zoomGesturesEnabled;
 
+  /// True if the map view should inclue a "My Location" dot
```

Comment:
Dart doc should start with a summary sentence followed by more details.
Maybe something like:
```
/// Whether to show a "My Location" layer over the map.
///
/// The layer includes a location indicator at the current device location if it is known.
/// The indicator is a small blue dot if the device is stationary, or a chevron if the device is moving.
```

We should also mention the permissions requirement, and explain what kind of setup is needed for this to work (e.g on Android edit their manifest, ask for permissions in runtime, on iOS edit Info.plist).

Does enabling this also shows the "Focus on current location" button? If so we should mention it in the Dartdoc as well.

Reply:
Ok for now the behavior is that both platforms add the My Location indicator _and_ button.